### PR TITLE
bsp: Updated ESP32 S3 LCD EV Board with new LVGL port and LVGL9 (BSP-504)

### PR DESCRIPTION
**Pull request by espzav**
# ESP-BSP Pull Request checklist
- [x] Version of modified component bumped
- [x] CI passing

# Change description
Updated BSP for ESP32 S3 LCD EV Board - now it uses LVGL port with LVGL9. 

- Removed these files due to incompatibility with LVGL9 - will be updated in #327 
- [ ] examples/display_audio_photo/sdkconfig.bsp.esp32_s3_lcd_ev_board
- [ ] examples/display_rotation/sdkconfig.bsp.esp32_s3_lcd_ev_board

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif @espressif2022 @pborcin PTAL
I tested only with screen 800x480, I haven't got 480x480 now. **Please** could anybody try it with 480x480 and tell me if is it working right? 
If not, I will do it on Tuesday next week.

**Comment by espressif2022** [comment]
> @tore-espressif @espressif2022 @pborcin PTAL I tested only with screen 800x480, I haven't got 480x480 now. **Please** could anybody try it with 480x480 and tell me if is it working right? If not, I will do it on Tuesday next week.

c0a56ca21e45d11a9dcc8efd97638506

I've tried it, and it's ok.

**Comment by espzav** [comment]
> I've tried it, and it's ok.
@espressif2022  Thank you very much!

File: bsp/esp32_s3_lcd_ev_board/include/bsp/esp32_s3_lcd_ev_board.h [hunk 1]
```diff
@@ -98,6 +111,7 @@ extern "C" {
  */
 typedef struct {
     void *dummy;    /*!< Prepared for future use. */
```
Comment (tore-espressif) [inline bsp/esp32_s3_lcd_ev_board/include/bsp/esp32_s3_lcd_ev_board.h:113]: Can we remove dummy now?

File: examples/display_audio_photo/sdkconfig.bsp.esp32_s3_lcd_ev_board [hunk 2]
```diff
@@ -1,27 +0,0 @@
-# This file was generated using idf.py save-defconfig. It can be edited manually.
```
[the remark this answers is not in the document]
Comment (espzav) [reply examples/display_audio_photo/sdkconfig.bsp.esp32_s3_lcd_ev_board:1]: I write it in description:
Removed these files due to incompatibility with LVGL9 - will be updated in #327 
- [ ] examples/display_audio_photo/sdkconfig.bsp.esp32_s3_lcd_ev_board
- [ ] examples/display_rotation/sdkconfig.bsp.esp32_s3_lcd_ev_board

File: bsp/esp32_s3_lcd_ev_board/include/bsp/esp32_s3_lcd_ev_board.h (hunk 1 above)
Comment (espzav) [reply bsp/esp32_s3_lcd_ev_board/include/bsp/esp32_s3_lcd_ev_board.h]: Removed.

File: examples/display_lvgl_demos/main/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,3 @@
-set(LV_DEMO_DIR ../managed_components/lvgl__lvgl/demos)
-file(GLOB_RECURSE LV_DEMOS_SOURCES ${LV_DEMO_DIR}/*.c)
-
 idf_component_register(
     SRCS "dispaly_lvgl_demos_main.c" ${LV_DEMOS_SOURCES}
```
[the remark this answers is not in the document]
Comment (espzav) [reply examples/display_lvgl_demos/main/CMakeLists.txt]: Removed.

**Comment by espzav** [comment]
@tore-espressif Thank you for the fast review :-)

**Review by tore-espressif — APPROVED** [review]
LGTM!

**Pull request merged**